waterloop-site/src/static/img/logos/Icon_Black.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="155" height="92" viewBox="0 0 155 92" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can you pass this SVG markup, as well as the other ones, through [SVGOMG](https://jakearchibald.github.io/svgomg/) to clean it up a bit and also prettify?
Also, could you move all SVGs to `waterloop-website-w20/waterloop-site/src/static/img/sustainable-tech`?